salt/orchestrate/bootstrap_step2.sls

```diff
@@ -0,0 +1,14 @@
+#  run_certs
+#  run_kubeconfig
+#  run_control_plane
+#  run_etcd
+#  run_mark_control_plane
+#  install_addons
+#  install_calico
+
+Bootstrap node:
+  salt.state:
+    - tgt: bootstrap
+    - saltenv: metalk8s-2.0
```

Comment:
After reflection don't know if it's really needed if not I suggest
```suggestion
    - saltenv: {{ saltenv }}
```

Reply:
Yep, it's needed.